api/types/backend/backend.go

```diff
@@ -40,6 +40,13 @@ type ContainerStatsConfig struct {
 	Version   string
 }
 
+// ContainerStatsAllConfig holds information for configuring the runtime
+// behavior of a backend.ContainerStatsAll() call.
+type ContainerStatsAllConfig struct {
+	ContainerStatsConfig
+	All bool
```

Comment:
Can we implement this as a filter, instead of a boolean `All`? `docker stats --all` can then set that filter (and in future we can add more filtering by adding a `--filter` flag)